server/db/migrate/20150408155047_remove_engine_hostname.rb

```diff
@@ -0,0 +1,9 @@
+class RenameEngineHostname < ActiveRecord::Migration
```

Comment:
The migration as written will currently fail with:  
```
/home/bbucking/.rvm/gems/ruby-1.9.3-p448@fortello/bin/ruby_executable_hooks:15:in `<main>'
caused by: (NameError) uninitialized constant RemoveEngineHostname
    ... skipped 41 lines
NameError: uninitialized constant RemoveEngineHostname
```

Need to update the class name to match the file name.

Regarding creating a migration for an engine, one way that it can be done, is to run the 'rails g migration' from the main app (foreman) and then move the migration files that are created over to the engine.